Create blogpost for release 2.22.0 (#49)
scm-manager/website: content/posts/2021-07-30-scm-manager-2-22-0/index.md

We also added powerful caching for internal working copies and fixed some bugs you reported.

## Omni Search - Details Page
Before you could search for repositories using the omni search in the header. Now you may still search for repositories shown as quick result, but also you can show `More results`. 
This leads to the new search details page which shows all search results categorized by result type. If you search for repository descriptions we also added highlighting inside the matching text fragments.

Tell us what you think about the new features and also stay excited since we just started to implement the code search, so you can search inside your repository files.

![Omni Search](assets/details-search.png)

## UI Redesign - Repository Overview

After the header, we have now redesigned the repository overview. We hope it is now clearer and easier to use.
We really need your opinion on this, so please give us feedback on how you like our changes.

![Repository Overview](assets/repository-overview.png)

## Caching for Working Directories

This feature is still experimental and a little hidden. Now you can activate caching for the internal working copies. 

> This is especially helpful if you work a lot with the [Editor plugin](https://scm-manager.org/plugins/scm-editor-plugin/) since it boosts the performance dramatically.

Reviewer: ```suggestion
This is especially helpful if you work a lot with the [editor plugin](https://scm-manager.org/plugins/scm-editor-plugin/) or the [review plugin](https://scm-manager.org/plugins/scm-review-plugin/), since it boosts the performance dramatically.
```

> Try it out using our [documentation](https://scm-manager.org/docs/2.22.x/en/administration/workdir_caching/).

Reviewer: ```suggestion
Try it out following our [documentation](https://scm-manager.org/docs/latest/en/administration/workdir_caching/).
```

## Other changes
- We fixed the file detection for hg repositories if directory and files were named the same

> - We decreased the log level of DefaultAdministrationContext from info to debug

Reviewer: ```suggestion
- We decreased the log level of `DefaultAdministrationContext` from info to debug
```

> - We use fixed the logo for the small header

Reviewer: ```suggestion
- We designed a new logo for the small header
```

> - We removed the repository short links on the overview since there were not used at all.

Reviewer: ```suggestion
- We removed the repository short links on the overview since they were not used at all.
```
